index.js

```diff
@@ -826,6 +888,18 @@
     return function(x) { return f(chain(x))(x); };
   }
 
+  //  Function$prototype$alt :: (x -> a) ~> (x -> a) -> x -> a
```

Comment:
`Function$prototype$alt :: (x -> a) ~> (x -> a) -> x -> a` here `a` is of kinde `*` but Alt is of kind `* -> *` so i think it should be `(Alt f) => (x -> f a) ~> (x -> f a) -> x -> f a` but not sure.